BLOG: editorial board updates-call for editors (#93)
pyOpenSci/pyopensci.github.io: _posts/2022-09-27-call-for-python-software-review-editors.md

---
layout: single
title: "pyOpenSci Editorial Process - New Editor in Chief and a Call For Editors"

> excerpt: "pyOpenSci is developing it's open peer review process for Python scientific software. Learn about our structure and if you are interested, apply to be on our editorial board."

Reviewer: ```suggestion
excerpt: "pyOpenSci is developing its open peer review process for Python scientific software. Learn about our structure and if you are interested, apply to be on our editorial board."
```
possesive its, no apostrophe

author: "Leah Wasser"
permalink: /blog/python-software-review-process-editor-in-chief
header:
    overlay_color: "#666"
    overlay_filter: 0.6
categories:
  - updates
  - blog-post
  - highlight
  - peer-review
---

## An update on pyOpenSci's review process 

Hey there! I just wanted to update everyone about where pyOpenSci is with 

> it's peer review process. We are currently hard at work updating [our peer 

Reviewer: ```suggestion
its peer review process. We are currently hard at work updating [our peer 
```
its not it's

> review guides](https://www.pyopensci.org/contributing-guide/intro.html) to 

Reviewer: suggest changing this to just "our guides" or "our guidebook" to avoid using the phrase "peer review" twice in the same sentence

streamline the peer review process. For the next 2 months (October & November 2022) 

> we will prioritize setting up a editorial board to support new reviews as they 

Reviewer: ```suggestion
we will prioritize setting up an editorial board to support new reviews as they 
```
an editorial board, in place of a editorial board

come in. 

With that below you will find a few updates.

## Welcome Editor in Chief David Nicholson 

[David Nicholson](https://github.com/NickleDave), who has been working with pyOpenSci for the past few years as 
an editor will take on the Editor in Chief role! [You can learn more about that 
role here](https://www.pyopensci.org/contributing-guide/open-source-software-submissions/editor-in-chief-guide.html) in our peer review guide. David will:

* Be the first person to reach out to anyone submitting a new review or a pre-submission. 

> * He will perform initial software checks to ensure your package is ready for  

Reviewer: ```suggestion
* Perform initial software checks to ensure your package is ready for  
```

> * He will also facilitate kicking off the peer review process by finding an editor for your package

Reviewer: ```suggestion
* Facilitate kicking off the peer review process by finding an editor for your package
```
Personal style ofc but suggest making the "one bullet point = one verb" consistent, or have them all be complete sentences

> David has a background behavioral / neuro / cognitive science and is a ML guru. 

Reviewer: ```suggestion
David has a background in behavioral / neuro / cognitive science and is a machine learning guru. 
```
:blush: thanks! can I put this on my resume :slightly_smiling_face:

Author: you should!

> Of course,  he's a Pythonista thru and thru. We are lucky to have David as a 

Reviewer: ```suggestion
Of course, he's a Pythonista through and through. We are lucky to have David as a 
```
(extra space)

(you can only get away with writing "thru and thru" if you are Keith Richards https://grammarist.com/spelling/through-and-through-or-thru-and-thru/ https://www.youtube.com/watch?v=8jbgPHAQVbk)

(and thank you)

Author: 😆

part of our pyOpenSci team. 

Stay tuned for a blog post from David introducing himself.

## We are creating an editorial board!

To support our open peer review, we want to bring on a few people (like yourself?!) 
to serve as guest editors / editors. Here's how this works:

Editors at pyOpenSci make a minimum of a 1-year commitment with a possibility
of longer given mutual consent. We also 
support guest editors in some instances. You will be responsible for handling 3-4 

> package reviews during that year. If you are new to our organization, we start 

Reviewer: The logical flow in this paragraph jumps around a bit. Maybe try this way?

> If you are new to our organization, we start you as a guest editor. What this means is that you will review one package and we will support you in that process to ensure you learn the ropes. Then if you are happy and work well with our editorial team we will bring you on officially as an editor. Editors at pyOpenSci make a minimum of a 1-year commitment with a possibility of longer given mutual consent. You will be responsible for handling 3-4 package reviews during that year. We may also bring on guest editors in other instances, e.g., because of their familiarity with a specific domain.

Author: added!! psss you can highlight multiple lines in a document and comment as well :) but i fixed this locally :)

you as a guest editor. What this means is that you will review one package and 
we will support you in that process to ensure you learn the ropes. Then if you 
are happy and work well with our editorial team we will bring you on as a 

> "full editor".

Reviewer: rewrite to avoid quotes? maybe
> we will bring you on officially as an editor

You can read more about the [roles and responsibilities of and editor here](https://www.pyopensci.org/contributing-guide/open-source-software-submissions/editors-guide.html).

### What background do I need to be an editor for pyOpenSci?

We welcome applications from potential editors with significant experience in 
one or more of the following areas: 
* open source software, 
* open science
* software engineering
* peer-review

> Members of the JOSS editorial team have diverse backgrounds. We welcome 

Reviewer: I'm a little confused about why we're talking about JOSS here

JOSS editors from academia, government, and industry. We especially welcome 
applications from prospective editors who will contribute to the diversity 
(ethnic, gender, disciplinary, and geographical) of our board. We also value 
having a range of junior and senior editors.

### How do I apply?

To sign up to be considered as an editor for pyOpenSci, please fill out the 
form below:

[Click here to apply to be an editor <i class="fa fa-4 fa-arrow-circle-right" aria-hidden="true"></i>](https://forms.gle/YH4kTeDFoYjDtefh7){: .btn .btn--info .btn--large}

## Questions?
If you have any questions, please reach out to me at leah at pyopensci.org or on 

> twitter at @leahawasser.

Reviewer: ```suggestion
twitter at [@LeahAWasser](https://twitter.com/LeahAWasser)```

* twitter can be case-sensitive when you try to @ user names
* might as well link to your twitter profile while you're at it
